src/IECoreMaya/ToMayaMeshConverter.cpp

```diff
@@ -344,6 +346,123 @@ bool ToMayaMeshConverter::doConversion( IECore::ConstObjectPtr from, MObject &to
 		}
 	}
 
+	/// Add corners and edge creases
+
+	MUintArray cornerIdsMaya;
+	MDoubleArray cornerSharpnessMaya;
+
+	const IECore::IntVectorData *cornerIdsData = mesh->cornerIds();
+	const std::vector<int> cornerIds = cornerIdsData->readable();
```

Comment:
Should be a reference? This seems to apply to the other calls to `someData->readable()` down below too...

Reply:
Embarassing. Fixed in 7f44cf8